src/app/pages/2_Sequence_Search.py

```diff
@@ -53,6 +57,7 @@
                     st.session_state[NCBI_DF] = ncbi.get_data(database, search_term)
                     data_processing.format_ncbi_summary()
                 except:
+                    logger.info(f"No results found for search query = '{search_term}'")
```

Comment:
I think `logger.warning` is probably better. Also, please add the `exc_info=e` argument. This will help surface/diagnose other errors that could occur during the search. Thx!

```suggestion
                    logger.warning(f"No results found for search query = '{search_term}'", exc_info=e)
```